examples/feeds/remove_feed_items.py

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env python
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Removes feed items from a feed."""
+
+
+import argparse
+import sys
+
+from google.ads.google_ads.client import GoogleAdsClient
+from google.ads.google_ads.errors import GoogleAdsException
+from google.ads.google_ads.util import ResourceName
+
+
+def main(client, customer_id, feed_id, feed_item_ids):
+    """The main method that creates all necessary entities for the example.
+
+    Args:
+        client: an initialized GoogleAdsClient instance.
+        customer_id: a client customer ID.
+        feed_id: the ID for a Feed belonging to the given customer.
+        feed_item_ids: a list of FeedItem IDs belonging to the given Feed.
+    """
+    feed_item_service = client.get_service("FeedItemService", version="v5")
+
+    operations = []
+    for feed_item_id in feed_item_ids:
+        # Constructs an operation that will remove the feed item based on the
+        # resource name.
+        feed_item_operation = client.get_type("FeedItemOperation", version="v5")
+        feed_item_operation.remove = feed_item_service.feed_item_path(
+            customer_id,
+            # constructs a resource name for a feed_item, which is in the
```

Comment:
This comment needs to be moved up two lines as `.feed_item_path` creates the full resource name whereas `.format_composite` only creates `{feed_id}~{feed_item_id}`

Reply:
Done